recipes/swig/all/conanfile.py

```diff
@@ -26,17 +26,25 @@ def _source_subfolder(self):
     def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
+    @property
+    def _use_pcre2(self):
+        return self.version not in ['4.0.1', '4.0.2']
```

Comment:
Maybe? not that another version _should_ get added but heck better safe then sorry

```suggestion
        return tools.Version(self.version) >= "4.1.0"
```